src/kms.ts

```diff
@@ -369,6 +381,67 @@ export class KmsClient {
     )
   }
 
+  /**
+   * Import a X509 certificate or a private key (both as DER encoded)
+   * @param {string} uniqueIdentifier  the unique identifier of the key
+   * @param {Uint8Array} derBytes the DER certificate/private key as bytes
+   * @param {string[]} tags potential list of tags
+   * @param {boolean} replaceExisting replace the existing object
+   * @param options Additional optional options
+   * @param {string} options.keyFormatType the key format type as specified in `object_data_structures.ts`
+   * @param {string} options.certificateIdentifier the private key id in order to create the link with the private key
```

Comment:
wrong phrase - certificate key id to create link with the private key

Reply:
thanks, don in a4855b7d4b9ca2562caa4bf2ac21b1153dd984cf